### PR TITLE
Add  more badges to readme

### PR DESCRIPTION
**Pull request by sawravchy**
- [x] Added New Badges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ Visual storytelling for WordPress.
 [![Commit activity](https://img.shields.io/github/commit-activity/m/google/web-stories-wp)](https://github.com/google/web-stories-wp/pulse/monthly)
 [![Code Coverage](https://codecov.io/gh/google/web-stories-wp/branch/main/graph/badge.svg)](https://codecov.io/gh/google/web-stories-wp)
 ![License](https://img.shields.io/github/license/google/web-stories-wp)
+[![GitHub repo size](https://img.shields.io/github/repo-size/google/web-stories-wp)](https://github.com/google/web-stories-wp)
+[![GitHub issues](https://img.shields.io/github/issues/google/web-stories-wp.svg)](https://GitHub.com/google/web-stories-wp/issues/)
+[![GitHub contributors](https://img.shields.io/github/contributors/google/web-stories-wp.svg)](https://GitHub.com/google/web-stories-wp/graphs/contributors/)
+
+
+
```
Comment (swissspidy) [inline README.md:14]: That's a lot of unnecessary whitespace

**Comment by swissspidy** [comment]
Thanks for your contribution!

Not sure how helpful all these badges are right now. I'll have to think about it and compare with some of our other repos.

**Comment by sawravchy** [comment]
> Thanks for your contribution!
> 
> Not sure how helpful all these badges are right now. I'll have to think about it and compare with some of our other repos.

Thanks.

File: README.md (hunk 1 above)
Comment (swissspidy) [inline README.md:14]:
```suggestion
[![GitHub repo size](https://img.shields.io/github/repo-size/google/web-stories-wp)](https://github.com/google/web-stories-wp)
[![GitHub issues](https://img.shields.io/github/issues/google/web-stories-wp.svg)](https://GitHub.com/google/web-stories-wp/issues/)
[![GitHub contributors](https://img.shields.io/github/contributors/google/web-stories-wp.svg)](https://GitHub.com/google/web-stories-wp/graphs/contributors/)
```

**Comment by swissspidy** [comment]
After looking at this again, I don't feel like adding these badges proves to be very useful.

I'm sure there might be some cases where adding some more badges to the readme might add value, but I think it's best to briefly discuss those in a small ticket beforehand.

For now I'll close this issue, even though I really appreciate you taking the time to submit this PR.
